### DISCUSSION TITLE
Numeric IDs

### DISCUSSION THREAD

**Discussion by Mailaender** [ClementJ18/moddb#50] [Q&A]
I saw that you scrape the numerical ID at

https://github.com/ClementJ18/moddb/blob/fe82ef91a355c4db597860be28ffc75629b4088f/moddb/pages/job.py#L73

Can I use it to link back to avoid using the URL slug?

So far I only so it being used in `https://button.moddb.com/rating/medium/mods/XXXXX.png` and for AJAX actions like adding to watch list.

**Comment by ClementJ18** [comment]
As far as I can tell, no. Slugs are the only way to access object pages.
